README.md

```diff
@@ -1,6 +1,10 @@
 # Official EdgeX Foundry Examples
 [![Go Report Card](https://goreportcard.com/badge/github.com/edgexfoundry/edgex-examples)](https://goreportcard.com/report/github.com/edgexfoundry/edgex-examples) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/edgex-examples)](https://github.com/edgexfoundry/edgex-examples/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/edgex-examples)](https://github.com/edgexfoundry/edgex-examples/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/edgex-examples-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/edgex-examples)](https://github.com/edgexfoundry/edgex-examples/commits)
 
+> **Warning**  
+> The **main** branch of this repository contains work-in-progress examples for the upcoming release, and is **not guaranteed to be stable or work with previous releases**.
+>
+> **Examples that work with the latest EdgeX release can be found at [Releases](https://github.com/edgexfoundry/edgex-examples/releases).** Navigate to the latest minor (x.y) tag for there.
```

Comment:
Does this sentence `Navigate to the latest minor (x.y) tag for there` needs to be modified?
```suggestion
> **Examples that work with the latest EdgeX release can be found at [Releases](https://github.com/edgexfoundry/edgex-examples/releases).** Navigate to the latest (x.y) tag.
```

Reply:
fixed. THX!